tests/codemods/test_flask_enable_csrf_protection.py

```diff
@@ -0,0 +1,97 @@
+from core_codemods.flask_enable_csrf_protection import FlaskEnableCSRFProtection
+from tests.codemods.base_codemod_test import BaseCodemodTest
+from textwrap import dedent
+
+
+class TestFlaskEnableCSRFProtection(BaseCodemodTest):
+    codemod = FlaskEnableCSRFProtection
+
+    def test_name(self):
+        assert self.codemod.name() == "flask-enable-csrf-protection"
+
+    def test_simple(self, tmpdir):
+        input_code = """\
+        from flask import Flask
+        app = Flask(__name__)
+        """
+        expected = """\
+        from flask import Flask
+        from flask_wtf.csrf import CSRFProtect
+
+        app = Flask(__name__)
+        csrf_app = CSRFProtect(app)
```

Comment:
If we don't change all subsequent references to `app`, then this change actually has no effect, right?